portfolio/src/main/java/com/google/sps/classes/LandingSight.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.classes;
+
+/** Represents a landing sight. */
+public class LandingSight {
+  private final double lat;
+  private final double lng;
+  private final String missionNum;
+  private final String description;
+
+  public LandingSight(double lat, double lng, String missionNum, String description) {
+    this.lat = lat;
+    this.lng = lng;
+    this.missionNum = missionNum;
+    this.description = description;
+  }
+}
```

Comment:
Gson is doing something sneaky here. Normally, there would be no way to access the fields of this class because they are all marked `private`. However, Gson is ignoring the visibility when marshaling to JSON. It's not a great idea to rely on this behavior, though, because [other common JSON libraries][] will not behave the same way -- you would just end up with an empty JSON object because all the fields are `private`.

While this technically works as-is, I would recommend making the fields of this class `public` just so that it is clear that they will be used outside of this class.

[other common JSON libraries]: https://github.com/FasterXML/jackson